script/upgrades/dependencies.json

```diff
@@ -1,10 +1,10 @@
 {
```

Comment:
It looks like the intention of this file is to store references to any dependencies that are not in our control. Contracts we are deploying as part of our updates should not be here. The addresses of any contracts we're deploying will be in the broadcast json files. These are loaded into the context of the script by first calling the helper function [contracts.load()](https://github.com/mento-protocol/mento-deployment/blob/971caff93f794079539bdb84c5134fb79c07e9eb/script/utils/Contracts.sol#L30) which takes the name of the deployment script and its timestamp as arguments. 

You can then pull the address for any contract that's been deployed as part of the loaded deployment scripts by calling [contracts.deployed()](https://github.com/mento-protocol/mento-deployment/blob/971caff93f794079539bdb84c5134fb79c07e9eb/script/utils/Contracts.sol#L74). This simplifies things as we always get the correct address and we do not have to keep updating this file. 

This may not be clear so it would be good to add this to the readme of the repo

Reply:
We just made a change to rename `mockUSDCet` to just `USDcet`. This is such that the deployment scripts can work on all networks. For Baklava and Alfajores, the address will be our deployed MockUSDcet contract, while in mainnet it will be the actual USDCet contract address from wormhole (therefore we need to have the addresses hardcoded in the dependencies.json file)